rootfs/scheduler/__init__.py

```diff
@@ -1049,6 +1051,44 @@ def _healthcheck(self, controller, routable=False, path='/', port=5000, delay=30
 
         return controller
 
+    def _readiness_probe(self, controller, delay=30, timeout=5, period_seconds=5,
+                         success_threshold=1, failure_threshold=1):
+        """
+        Apply exec readiness check to the slugrunner container
+
+        http://kubernetes.io/docs/user-guide/pod-states/#container-probes
+        """
+        readinessprobe = {
+            'readinessProbe': {
+                # an exec probe
+                'exec': {
+                    "command": [
+                        "bash",
+                        "-c",
+                        "[[ '$(ps -p 1 -o args)' != *'bash /runner/init'* ]]"
+                    ]
+                },
+                # length of time to wait for a pod to initialize
+                # after pod startup, before applying health checking
```

Comment:
You can probably just hardcode these values instead of having it configurable via the method signature - Do you think we will / should ever made it usable anywhere else?